start-ocp-console.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+CONSOLE_VERSION=${CONSOLE_VERSION:=4.14}
+CONSOLE_PORT=${CONSOLE_PORT:=9000}
+CONSOLE_IMAGE="quay.io/openshift/origin-console:${CONSOLE_VERSION}"
+
+echo "Starting local OpenShift console..."
+
+BRIDGE_BASE_ADDRESS="http://localhost:9000"
+BRIDGE_USER_AUTH="disabled"
+BRIDGE_K8S_MODE="off-cluster"
+BRIDGE_K8S_AUTH="bearer-token"
+BRIDGE_K8S_MODE_OFF_CLUSTER_SKIP_VERIFY_TLS=true
+BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT=$(oc whoami --show-server)
+BRIDGE_PLUGINS="odf-console=http://localhost:9001"
```

Comment:
Let's read the active plugins from environment:
```suggestion
BRIDGE_PLUGINS=${BRIDGE_PLUGINS:=odf-console=http://localhost:9001}
```

Reply:
https://github.com/red-hat-storage/odf-console/pull/972#issue-1852974243